skylark_library.bzl

```diff
@@ -1,109 +1,7 @@
-# Copyright 2017 The Bazel Authors. All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+load("//:bzl_library.bzl", "StarlarkLibraryInfo", "bzl_library")
 
-"""Skylib module containing a library rule for aggregating rules files."""
+# These are temporary forwarding macros to facilitate migration to
+# the new names for these objects.
+SkylarkLibraryInfo = StarlarkLibraryInfo
```

Comment:
I always thought you had to do the import and alias it with an underscore, this works?  It knows they are different things?

Reply:
IIRC rules, providers, and aspects are given their names at the end of execution of the file in which they're defined (so where the actual `provider` function is called), and any aliasing after that just creates a new reference to the old thing.

Reply:
Doh, I was reading those as the same, missed the whole _Sky_ vs _Star_.  Ignore me.